tutor/src/screens/assignment-review/drop-question-modal.tsx

```diff
@@ -19,52 +21,101 @@ interface DropQuestionModelProps {
     ux: UX
 }
 
+const Light = styled.div`
+font-size: 1.2rem;
+color: #6F6F6F;
```

Comment:
we might have something like this elsewhere but if not we should think about making a file of common things like this

Reply:
Yeah, it'd be nice to build out some common styled components that handle common typography